### PR TITLE
[feature-wip](unique-key-merge-on-write) remove AggType on unique table with MoW, enable preAggreation, DSIP-018[5/2]

### PR DESCRIPTION
**Pull request by zhannngchen**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

see DSIP-018:https://cwiki.apache.org/confluence/display/DORIS/DSIP-018%3A+Support+Merge-On-Write+implementation+for+UNIQUE+KEY+data+model
This is the patch for stage 5-2

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/memtable.cpp [hunk 1]
```diff
@@ -64,8 +64,8 @@ MemTable::MemTable(int64_t tablet_id, Schema* schema, const TabletSchema* tablet
         } else {
             _insert_fn = &MemTable::_insert_agg;
         }
-        if (_tablet_schema->has_sequence_col()) {
-            _aggregate_two_row_fn = &MemTable::_aggregate_two_row_with_sequence;
```
Comment (pengxiangyu) [inline be/src/olap/memtable.cpp:68]: Is sequence_col in tablet_schema not needed？what abort old meta info？

Comment (zhannngchen) [reply be/src/olap/memtable.cpp:68]: This part of code is not removed, just move to `_aggregate_two_row`, it works quite similar like the method `_aggregate_two_row_in_block`.
For the unique table without MoW(merge-on-write), they all proceeded by the method `_aggregate_two_row`, no necessary to separate to 2 methods.

**Comment by zhannngchen** [comment]
@EmmyMiao87 can you help to review the code change of Planer?
Basically,  when user enabled unique key merge-on-write, we don't need to merge the data with same keys to ensure which row is latest, old row is marked as deleted in DeleteBitmap. So we can enable pre-aggregation to accelerate the query execution, just like processing data in duplicate keys table.

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**